idearium-lib/test/lib-mq-connection.js

```diff
@@ -58,7 +58,18 @@ describe('class mq.Connection', function () {
 
     describe('gracefully disconnects', function () {
 
-        it('RabbitMQ', function (done) {
+        it('when not connected', function (done) {
+
+            // Setup an instance of the class.
+            const ideariumMq = new mq.Connection(conf.rabbitUrl);
+
+            ideariumMq.disconnect()
+                .then(done)
```

Comment:
@smebberson Just a question here, does the disconnect ever return anything? Wouldn't it be safer to have `.then(() => done())`

Reply:
@allanchau, good call. Just updated.